web/src/main/java/org/springframework/security/web/header/writers/PermissionsPolicyHeaderWriter.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2002-2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.security.web.header.writers;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.springframework.security.web.header.HeaderWriter;
+import org.springframework.util.Assert;
+
+/**
+ * Provides support for
+ * <a href="https://w3c.github.io/webappsec-permissions-policy//">Permisisons Policy</a>.
+ * <p>
+ * Permissions Policy allows web developers to selectively enable, disable, and modify the
+ * behavior of certain APIs and web features in the browser.
+ * <p>
+ * A declaration of a permissions policy contains a set of security policies, each
+ * responsible for declaring the restrictions for a particular feature type.
+ *
+ * @author Christophe Gilles
+ * @since 5.5
+ */
+public final class PermissionsPolicyHeaderWriter implements HeaderWriter {
+
+	private static final String PERMISSIONS_POLICY_HEADER = "Permissions-Policy";
+
+	private String policy;
+
+	/**
+	 * Create a new instance of {@link PermissionsPolicyHeaderWriter}.
+	 */
+	public PermissionsPolicyHeaderWriter() {
+		this("");
```

Comment:
I don't think this should set the policy to "". 
We want to ensure that users are configuring the policy if they are using the `PermissionsPolicyHeaderWriter`.
This will allow them to create a `PermissionsPolicyHeaderWriter` without specifying the policy.

Reply:
Ok so we should have two ways to set the Policy ? With String and with Lambda ?

Reply:
Yes
```
http
	.headers()
		.permissionsPolicy()
			.policy("geolocation 'self'");
```
and 
```
http
	.headers(headers -> headers
		.permissionsPolicy(permissions -> permissions
			.policy("geolocation 'self'")
		)
	);
```

Reply:
Ok, I change constructor and add some tests to be sure everything is ok